Frameworks/CoreText/CTFont.mm

```diff
@@ -508,19 +601,19 @@ CGFloat CTFontGetUnderlineThickness(CTFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CGFloat CTFontGetSlantAngle(CTFontRef font) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return _DWriteFontGetSlantDegrees(font->_dwriteFontFace);
```

Comment:
Testing on the ref plat:

CTFontGetSlantAngle(NULL) returns 0

however

CFDictionaryGetValue(NULL, CFSTR("FOO")) raises an AV, indicating a general principle of this being okay

I don't think this is a situation where we care super-much about adhering to the former rather than the latter - probably an arbitrary decision by a lone dev.

---
In reply to: [80783054](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80783054) [](ancestors = 80783054,80757662,80733021)